### ISSUE TITLE
Serialization bug with UploadImageUrlAsync

### ISSUE THREAD

**Issue by nomorechickennuggets** [DamienDennehy/Imgur.API#14]
Encountering serialization error with the following:

                    var endpoint = new ImageEndpoint(client);
                    var image = await endpoint.UploadImageUrlAsync(photo.Url); // exception thrown here with known good URL

      
```
   Newtonsoft.Json.JsonReaderException was caught  
  _HResult=-2146233088
  _message=Unexpected character encountered while parsing value: {. Path 'data.error', line 1, position 18.
  HResult=-2146233088
  IsTransient=false
  Message=Unexpected character encountered while parsing value: {. Path 'data.error', line 1, position 18.
  Source=Newtonsoft.Json
  LineNumber=1
  LinePosition=18
  Path=data.error
```

**Comment by DamienDennehy** [comment]
A few months ago Imgur changed the structure of some of the errors they return from their API.
I fixed some of them in the 5.0.0-beta1 release, try using that in the meantime.

**Comment by DamienDennehy** [comment]
Resolved in v5.
